### PR TITLE
Add the timezone as fetched from API data

### LINKED ISSUE #707: Call for Speakers: Time and Timezone issue
The set timing in the call for speakers sections should adhere to the following:
* Set prefilled beginning time to before the event (if possible and event is not schedule for tomorrow)
* Use the timezone that was set in wizard step 1

### PR DESCRIPTION
**Pull request by sumedh123**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
Fixes #707

### CODE REVIEW

**Comment by sumedh123** [comment]
@fossasia/open-event-frontend Please review. In the copy link issue, I missed one thing. Hence, corrected in this one.

**Comment by niranjan94** [comment]
Deployment link: https://pr-721-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@CosmicCoder96 That wont be required here as the CFS is last step and one can only reach it if step 1 is complete where the timezone & dates are set. 

@niranjan94 Do you think we should disable the change as in the tickets settings?

**Comment by sumedh123** [comment]
@CosmicCoder96 I think the server side returns the timezone automatically. Still I made the changes. Please review.

**Comment by niranjan94** [comment]
CFS timezone should not be editable. Should use event timezone.

**Comment by niranjan94** [comment]
And event timezone must be selected. Default that user's timezone. If that isn't available `UTC`

**Comment by geekyd** [comment]
@CosmicCoder96 That wont be required here as the CFS is last step and one can only reach it if step 1 is complete where the timezone & dates are set. 

@niranjan94 Do you think we should disable the change as in the tickets settings?

**Comment by niranjan94** [comment]
@geekyd yes. It should be disabled. I remember it was disabled at one point.

**Comment by niranjan94** [comment]
Deployment link: https://pr-721-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-721-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@geekyd @CosmicCoder96 Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-721-open-event-frontend-fossasia.surge.sh

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 1]
```diff
@@ -149,10 +149,10 @@
       </div>
       <div class="five wide field">
```
Comment (geekyd) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: @sumedh123 Remove the ui-dropdown it is not needed use the implementation from ticket-input instead
```
<div class="five wide field has popup" data-content="Uses the timezone of the event.">
  <label class="required" for="ticket_timezone">{{t 'Timezone'}}</label>
  {{input type='text' disabled=true value=timezone id='ticket_timezone' }}
</div>
```

**Comment by sumedh123** [comment]
@geekyd Done.

**Comment by niranjan94** [comment]
Deployment link: https://pr-721-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-721-open-event-frontend-fossasia.surge.sh

**Pull request merged**